src/main/kotlin/net/leanix/githubagent/handler/BrokerStompSessionHandler.kt

```diff
@@ -38,11 +42,23 @@ class BrokerStompSessionHandler : StompSessionHandlerAdapter() {
     }
 
     override fun handleTransportError(session: StompSession, exception: Throwable) {
-        logger.error("handleTransportError", exception)
+        logger.error("Connection error")
```

Comment:
Suggestion
```suggestion
        logger.error("Connection error: ${exception.message}")
```

Reply:
This is a `exception: Throwable` and it does not hurt to have the message in the logs. I understand not wanting the stack trace in the logs but the message could be helpful when debugging an issue.